### DISCUSSION TITLE
Dynamic Sheets Name

### DISCUSSION THREAD

**Discussion by calvinliem** [eclipse-birt/birt#1773] [Q&A]
Hello, everyone. I would like to ask how to do Dynamic Sheets name for excel export in BIRT Designer.
My condition is, i have a table that had an table inside (with Grouping) like this

>Table 
          >Table (with Grouping)
          
this is what my structure LIKE
![image](https://github.com/eclipse-birt/birt/assets/57037399/3ca4dd03-f3e5-4e32-a82b-0b83d0c2af2b)

**Comment by SteveSchafer-Innovent** [comment]
I just did a little experimenting with this and it looks like, if you output to xlsx, it will form a new sheet with every page break.  I found that specifying a page break in the group doesn't work, but adding a page break to a group header row does.  I believe the sheet names come from the table-of-contents setting in the group.
